streaming/index.js

```diff
@@ -16,6 +16,8 @@ dotenv.config({
   path: env === 'production' ? '.env.production' : '.env',
 });
 
+log.level = process.env.LOG_LEVEL || 'verbose';
```

Comment:
Ensure that log level is set at start of executing, rather than only when http/ws server starts listening